apps/github-actions-cache/docker-compose.dev.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.3'
+
+services:
+  redis-cluster:
+    container_name: ads_redis_cluster
```

Comment:
ads is a short for the air-discount-scheme project.

Reply:
renamed